### ISSUE TITLE
Sortable Component issues with ng-template

### ISSUE THREAD

**Issue by cp79shark** [telerik/kendo-angular#914]
## I'm submitting a...

* Bug report 

## Current behavior

The Sortable component (it) does not seem to work really at all with Templates. Yes, I've read "Known Limitations" and looked through all the docs and issues. 

- It will not start the drag operation on the first click, unless it's certain elements. (for example, initiate the drag by clicking the top level div instead of the span or child div)
- It will not inititate the dragEnd event, unless it's certain element initiating drag start.
- It will not set the DragDropEvent Index properly for dragEnd unless it's certain elements initiating dragStart AND the drop occurs outside the container.

It's entirely possible I'm doing something "dumb". I would love that to be the case, but I don't see what that could be with this simple of a setup.

## Expected behavior

The expected behavior is it works with templates and I get the correct index on dragEnd always.

## Minimal reproduction of the problem with instructions

[Plunkr](https://plnkr.co/edit/LTkKK2tRmAHbyult8bsj?p=preview)

My connection is pretty terrible and I wasn't sure how to setup the Plunkr for pulling in the kendo libs, but the Plunkr "works" and shows what I'm talking about better than I probably described it above.

## What is the motivation or use case for changing the behavior?

Motivation is the control should work.

## Environment

Package versions:

```
See below
```

Browser:

- Chrome (desktop) version  60.0.3112.113
- IE version 11

System:
Just going to copy my config.js from the Plunkr.
```
System.config({
  //use typescript for compilation
  transpiler: 'typescript',
  //typescript compiler options
  typescriptOptions: {
    emitDecoratorMetadata: true
  },
  paths: {
    'npm:': 'https://unpkg.com/'
  },
  //map tells the System loader where to look for things
  map: {
    "@progress": "./src",
    'app': './src',
    '@angular/core': 'npm:@angular/core'+ angularVersion + '/bundles/core.umd.js',
    '@angular/common': 'npm:@angular/common' + angularVersion + '/bundles/common.umd.js',
    '@angular/common/http': 'npm:@angular/common' + angularVersion + '/bundles/common-http.umd.js',
    '@angular/compiler': 'npm:@angular/compiler' + angularVersion  + '/bundles/compiler.umd.js',
    '@angular/platform-browser': 'npm:@angular/platform-browser' + angularVersion + '/bundles/platform-browser.umd.js',
    '@angular/platform-browser-dynamic': 'npm:@angular/platform-browser-dynamic' + angularVersion + '/bundles/platform-browser-dynamic.umd.js',
    '@angular/http': 'npm:@angular/http' + angularVersion + '/bundles/http.umd.js',
    '@angular/router': 'npm:@angular/router' + angularVersion +'/bundles/router.umd.js',
    '@angular/forms': 'npm:@angular/forms' + angularVersion + '/bundles/forms.umd.js',
    '@angular/animations': 'npm:@angular/animations' + angularVersion + '/bundles/animations.umd.js',
    '@angular/platform-browser/animations': 'npm:@angular/platform-browser' + angularVersion + '/bundles/platform-browser-animations.umd.js',
    '@angular/animations/browser': 'npm:@angular/animations' + angularVersion + '/bundles/animations-browser.umd.js',
    
    '@angular/core/testing': 'npm:@angular/core' + angularVersion + '/bundles/core-testing.umd.js',
    '@angular/common/testing': 'npm:@angular/common' + angularVersion + '/bundles/common-testing.umd.js',
    '@angular/common/http/testing': 'npm:@angular/common' + angularVersion + '/bundles/common-http-testing.umd.js',
    '@angular/compiler/testing': 'npm:@angular/compiler' + angularVersion + '/bundles/compiler-testing.umd.js',
    '@angular/platform-browser/testing': 'npm:@angular/platform-browser' + angularVersion + '/bundles/platform-browser-testing.umd.js',
    '@angular/platform-browser-dynamic/testing': 'npm:@angular/platform-browser-dynamic' + angularVersion + '/bundles/platform-browser-dynamic-testing.umd.js',
    '@angular/http/testing': 'npm:@angular/http' + angularVersion + '/bundles/http-testing.umd.js',
    '@angular/router/testing': 'npm:@angular/router' + angularVersion + '/bundles/router-testing.umd.js',
    'tslib': 'npm:tslib@1.6.1',
    'rxjs': 'npm:rxjs',
    'typescript': 'npm:typescript@2.2.1/lib/typescript.js'
  },
  //packages defines our app package
  packages: {
    app: {
      main: './main.ts',
      defaultExtension: 'ts'
    },
    rxjs: {
      defaultExtension: 'js'
    },
      '@progress/kendo-angular-l10n': {
      "main": "./kendo-angular-l10n.js",
      "defaultExtension": "js"
      },    
      '@progress/kendo-angular-sortable': {
      "main": "./kendo-angular-sortable.js",
      "defaultExtension": "js"
      }, 

  }
});
```

**Comment by tapopov** [comment]
Hello @cp79shark 

Thanks for the example. I reviewed it and noticed that the draggable attribute is set to each element, which isn't really necessary. Removing that attribute seem to fix the issue where the container is not dragged on the first click.
The issue with the wrong index on dragEnd event is a known one and is logged [here](https://github.com/telerik/kendo-angular/issues/550). All of those should be solved in the upcoming 2.0 version, which uses a rather different approach to handling the events and mutating the data. Here's a [GitHub issue](https://github.com/telerik/kendo-angular/issues/828) related to the planned changes.

**Comment by cp79shark** [comment]
@tapopov, thanks for the response. Yeah, I had left that attribute on a whole slew of elements as a "last-gasp" effort. Thought I did have it set on none as a test, but the Plunkr says otherwise. Can confirm initial drag is fine (though really I'd want draggable="true" only set on a certain element in the template, the "drag handle").

I'll subscribe to 828 and just move on with Dragula for the time being until the updated version is released due the drag index issue.
